improver/blending/weights.py

```diff
@@ -640,7 +640,7 @@ def process(self, cube, coord_name, inverse_ordering=False):
             # make a copy of the input cube from which to calculate weights
             inverted_cube = cube.copy()
             inverted_cube = sort_coord_in_cube(
-                inverted_cube, coord_name, order="descending")
+                inverted_cube, coord_name, True)
```

Comment:
It is much clearer what is True if you name the keyword. Can you do this throughout please.
```suggestion
                inverted_cube, coord_name, descending=True)
```